app/create_file.py

```diff
@@ -1 +1,95 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+
+def create_directory(path: str) -> None:
+    try:
+        os.makedirs(path)
+        print(f"Created directory: {path}")
+    except FileExistsError:
+        print(f"Directory already exists: {path}")
+    except OSError as e:
+        print(f"Failed to create directory: {path}")
+        print(f"Error: {str(e)}")
```

Comment:
Just simplify code with:
```suggestion
        os.makedirs(path, exist_ok=True)
        print(f"Created directory: {path}")
```